packages/mui-material/src/Modal/Modal.test.js

```diff
@@ -599,15 +598,13 @@ describe('<Modal />', () => {
 
           return (
             <Modal open={open}>
-              <Fade in={open} timeout={timeout}>
-                <div id="modal-body">hello</div>
-              </Fade>
+              {/* TODO: Look into why this test started to fail with React 19 when using a transition component as children. */}
+              {/* See: https://github.com/mui/material-ui/issues/43312 */}
+              <div id="modal-body">hello</div>
```

Comment:
@mnajdova we talked about skipping this test until but removing the transition component makes it pass. It's not ideal but at least we keep testing a simpler use case. wdyt?

Reply:
Yep, let's do it!

Reply:
Removing the test seems to defeat the purpose of being compatible with React 19. If it doesn't pass, it sounds like it's not compatible.

We could only use React 19 for the docs, but stick to 18 for the tests, until we solve this one.

Reply:
> We could only use React 19 for the docs, but stick to 18 for the tests, until we solve this one.

This is one test out of more than 8 thousand that run correctly, I strongly prefer running the tests in the latest React version. I created [an issue](https://github.com/mui/material-ui/issues/44674) to continue this investigation, but it shouldn't block the bump of the packages' React version.